src/main/java/utils/Account.java

```diff
@@ -0,0 +1,13 @@
+package utils;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter @Setter
```

Comment:
There is also cool lombok annotation called `@Data` which includes setters and getters 
https://projectlombok.org/features/Data